libr/bin/format/elf/glibc_elf.h

```diff
@@ -3964,6 +3964,10 @@ enum
 
 #define R_BPF_NONE		0	/* No reloc */
 #define R_BPF_64_64		1
+#define R_BPF_64_ABS64		2
+#define R_BPF_64_ABS32		3
+#define R_BPF_64_NODYLD32	4
+#define R_BPF_64_RELATIVE	8
```

Comment:
`R_BPF_64_RELATIVE` is a really odd one.

It applies on the ldxdw instruction (read "load 64-bit value from absolute 64-bit address").
This instruction is encoded using 128 bits (four 32-bit words). The first word stores the virtual address (ELF file offset) to resolve. After resolving it to a physical address, the high word of the physical address gets stored in the first word and the low word in the third word.

I don't think `RBinReloc` can express this behavior.

Reply:
Pre-relocation
```
[ 127..96 ] [ 95..64 ] [ 63..32 ] [ 31..0 ]
              ^ virtual address bits 31..0
```
Post-relocation
```
[ 127..96 ] [ 95..64 ] [ 63..32 ] [ 31..0 ]
              ^ physical address bits 63..32 (i think??)
                                    ^ physical address bits 31..0
```